presto-docs/src/main/sphinx/sql/analyze.rst

```diff
@@ -15,7 +15,9 @@ Description
 Collects table and column statistics for a given table.
 
 The optional ``WITH`` clause can be used to provide
-connector-specific properties.
+connector-specific properties. To list all available properties, run the following query::
```

Comment:
The original wrapping here is odd (likely overzealous in wrapping at 80 characters). Let's format like
```
The optional ``WITH`` clause can be used to provide connector-specific properties.
To list all available properties, run the following query::

Reply:
Addressed as fixup.